plugins/woocommerce/src/Internal/Abilities/REST/RestAbility.php

```
@@ -0,0 +1,37 @@
+	/**
+	 * Skip output validation for REST abilities.
+	 *
+	 * WooCommerce REST API schemas often don't accurately reflect the actual
+	 * output, particularly for nullable fields and type variations. Rather than
+	 * trying to fix all schema inconsistencies, we skip output validation for
+	 * REST-based abilities while maintaining input validation and permissions.
+	 *
+	 * @param mixed $output The output to validate.
+	 * @return true Always returns true (no validation).
+	 */
+	protected function validate_output( $output ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$output'. (undefined)

(UnusedFormalParameter)